packages/enzyme-adapter-utils/src/Utils.js

```diff
@@ -168,9 +178,20 @@ export function ensureKeyOrUndefined(key) {
 }
 
 export function elementToTree(el) {
-  if (el === null || typeof el !== 'object' || !('type' in el)) {
+  if (el === null || typeof el !== 'object' || !('type' in el || el.$$typeof === Portal)) {
```

Comment:
the `$$typeof` part should definitely not be here; that should be adapter-specific.